etc/module.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Razorpay_Magento" setup_version="3.8.5-beta">
+    <module name="Razorpay_Magento" setup_version="3.8.4-beta">
```

Comment:
@nikhilsharma1100 If required raise separate PR for version bump. Please avoid including version bump with enhancement OR fix PR.

Reply:
This is no version bump in this PR. It is showing this change because I raised PR from another branch which had this beta version while testing.